tcms/testplans/static/testplans/js/search.js

```diff
@@ -172,6 +170,24 @@ export function pageTestplansSearchReadyHandler () {
     $('#id_product').change(updateVersionSelectFromProduct)
 }
 
+function hideExpandedChildren (table, parentRow) {
+    const children = hiddenChildRows[parentRow.data().id]
+    if (!children) {
+        return
+    }
```

Comment:
I think this is redundant. When `children` is empty the loop below will not iterate and the function will exit anyway.

Reply:
@atodorov 
If `parentRow.data().id` not in `hiddenChildRows` childer will be `undefined` and then it will crash.

I don't know if this is a possible situation, what do you think?

Reply:
> If parentRow.data().id not in hiddenChildRows

That's not supposed to happen, see line 129-135. That would mean you are trying to collapse a row which has no children to begin with, so it shouldn't be expanded in the first place.  If that happens there is probably a bug somewhere else, not in the `hideExpandedChildren()` function. Let it crash if that's the case so we can see it and fix it.

Reply:
Fair. Fixed